src/components/editions/article.tsx

```diff
@@ -0,0 +1,62 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css } from '@emotion/core';
+import { border } from '@guardian/src-foundations/palette';
+import { remSpace } from '@guardian/src-foundations';
+import { Lines } from '@guardian/src-ed-lines';
+import { Design } from '@guardian/types/Format';
+import { partition } from '@guardian/types/result';
+
+import { Item } from 'item';
+import { renderAll } from 'renderer';
+import Standfirst from 'components/editions/standfirst';
+import HeaderImage from 'components/editions/headerImage';
+import Series from 'components/editions/series';
+import Headline from 'components/editions/headline';
+import Byline from 'components/editions/byline';
+
+
+// ----- Component ----- //
+
+interface Props {
+    item: Item;
```

Comment:
Item seems a strange name but assume its something legacy?

Reply:
It's not legacy, it's probably the most important type in the codebase 😄. It refers to a CAPI "item" - you can see the same terminology used, for example, [in MAPI](https://github.com/guardian/mobile-apps-api/tree/master/mobile-item).